services/notifications/pkg/service/service.go

```diff
@@ -126,6 +127,18 @@ func (s eventsNotifier) send(ctx context.Context, recipientList []*channels.Mess
 	}
 }
 
+func (s eventsNotifier) ensureGranteeList(ctx context.Context, executant, u *user.UserId, g *group.GroupId) []*user.User {
+	if granteeList, err := s.getGranteeList(ctx, executant, u, g); err != nil {
+		s.logger.Error().Err(err).Str("event", "ensureGranteeList").Msg("Could not get grantee list")
+		return nil
+	} else if len(granteeList) == 0 {
+		s.logger.Debug().Str("event", "ensureGranteeList").Msg("The grantee list is empty")
+		return nil
+	} else {
```

Comment:
Please remove the last else and return `granteeList` in main thread

Reply:
ok